.gitlab/dev_container_deploy/docker_linux.yml

```diff
@@ -276,20 +276,30 @@ qa_agent:
   needs:
     - docker_build_agent7
     - docker_build_agent7_arm64
-    - docker_build_ot_agent7
-    - docker_build_ot_agent7_arm64
     - docker_build_agent7_windows1809
     - docker_build_agent7_windows2022
   variables:
     IMG_REGISTRIES: agent-qa
-  parallel:
-    matrix:
-      - IMG_SOURCES: ${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-arm64
-        IMG_DESTINATIONS: agent:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}
-      - IMG_SOURCES: ${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-win1809-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-winltsc2022-amd64
-        IMG_DESTINATIONS: agent:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}
-      - IMG_SOURCES: ${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot-arm64
-        IMG_DESTINATIONS: agent:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-ot
+    IMG_SOURCES: ${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-arm64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-win1809-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-7-winltsc2022-amd64
+    IMG_DESTINATIONS: agent:${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}
+
+qa_agent_ot:
+  extends: .docker_publish_job_definition
+  stage: dev_container_deploy
+  rules:
+    - !reference [.on_aml_or_e2e_changes]
+    - !reference [.on_apm_or_e2e_changes]
+    - !reference [.on_container_or_e2e_changes]
+    - !reference [.on_npm_or_e2e_changes]
+    - !reference [.on_process_or_e2e_changes]
```

Comment:
You probably do not need these rules because the `agent_ot` image is not used by the existing e2e tests.
Later on, when some tests start to use `agent_ot` you will need to update these rules to make sure the image is published when a test that needs the image is executed